components/TaskRecommendation.tsx

```diff
@@ -0,0 +1,83 @@
+import ClayButton from '@clayui/button';
+import ClayIcon from '@clayui/icon';
+import ClayLayout from '@clayui/layout';
+import ClayPanel from '@clayui/panel';
+import React from 'react';
+import ReactMarkdown from 'react-markdown';
+
+import CodeBlock from './CodeBlock';
+
+const TaskRecommendation = ({
+	comment,
+	comments,
+	handleStagedChanges,
+	index,
+	isStaged,
+	postStaged,
+	stagedChanges,
+}) => {
+	return (
+		<>
+			<ClayPanel.Header>
+				<ClayLayout.ContentRow containerElement="h3" float>
+					<ClayLayout.ContentCol>
+						{`Line ${comment.line}: ${comment.title}`}
+					</ClayLayout.ContentCol>
+					<ClayLayout.ContentCol expand>
+						<ClayButton
+							// @ts-ignore
+							displayType={isStaged ? 'success' : 'secondary'}
+							onClick={() => {
+								postStaged(!isStaged, comment.id);
+
+								const newArray = [...stagedChanges];
+
+								if (isStaged) {
+									newArray.splice(
+										newArray.indexOf(comment.id),
+										1
+									);
+								} else {
+									newArray.push(comment.id);
+								}
+
+								handleStagedChanges(newArray);
+							}}
+							small
+							style={{
+								marginLeft: 'auto',
+							}}
+						>
+							{isStaged ? 'Staged' : 'Stage Change'}
+
+							{isStaged && (
+								<ClayIcon
+									symbol="check"
+									style={{
+										marginLeft: 4,
```

Comment:
instead of using inline styles, you can use `ml-1`. See https://clayui.com/docs/css/utilities/c-spacing.html

> 1 - 0.25rem - 4px

Reply:
Good catch! 👍